### PR TITLE
ENH: added parse_blocking_func function that accepts strings as input

### PR DESCRIPTION
**Pull request by mbaak**

### CODE REVIEW

**Review by faab5 — APPROVED** [review]

**Review by faab5 — APPROVED** [review]

File: emm/helper/blocking_functions.py [hunk 1]
```diff
@@ -37,3 +41,27 @@ def first2(x: str) -> str:
 def first3(x: str) -> str:
     """First two characters blocking function."""
     return x.strip().lower()[:3]
+
+
+BLOCKING_MAP = {"first": first, "first2": first2, "first3": first3}
+
+
+def parse_blocking_func(input: Callable[[str], str] | str | None = None):
```
Comment (sbrugman) [inline emm/helper/blocking_functions.py:49]: Return type

Comment (sbrugman) [inline emm/helper/blocking_functions.py:49]: Make function private (`_parse_blocking_func`)

Comment (mbaak) [reply emm/helper/blocking_functions.py]: Added.

Comment (mbaak) [reply emm/helper/blocking_functions.py]: Done.

**Review by sbrugman — APPROVED** [review]

**Pull request merged**